bot.ml

```diff
@@ -224,16 +219,17 @@ let handle_json action default body =
       Stdio.printf "Json type error: %s\n" err ;
       default
 
-let generic_get relative_uri ?(header_list = []) ~default json_handler =
+let generic_get relative_uri ~header_list ~default json_handler =
   let uri = "https://api.github.com/" ^ relative_uri |> Uri.of_string in
```

Comment:
Despite the `generic_get` name, this function is specific to the GitHub API, so it could already be configured with the `github_header` by default.